rpm/pacemaker.spec.in

```diff
@@ -517,7 +517,7 @@ popd
 %check
 make %{_smp_mflags} check
 { cts/cts-scheduler --run load-stopped-loop \
-  && cts/cts-cli \
+  && TMPDIR=%{_buildrootdir} cts/cts-cli -V \
```

Comment:
as long as the `-V` flag's output changes don't break anything (further)... and you wouldn't think so, given that `cts-cli` itself should be the only thing checking output AFAIK

In Slack, Fabio was saying BUILDDIR and then "or $(build_dir)/ in autotools world". I found this:
https://docs.fedoraproject.org/en-US/packaging-guidelines/RPMMacros/#_macros_set_for_the_rpm_and_srpm_build_process

I haven't looked into the roles of all these RPM build directories. Is there a particular reason for choosing the `BUILDROOT` directory over some other like `BUILD` or `%{buildroot}` -- or arbitrary?

Reply:
Fabio also suggested -- and it seems like a reasonable idea -- updating `cts-cli.in` itself to default to something other than `/tmp`, if `TMPDIR` is not set. Something in the PWD or source dir or whatever; something that we "own."

Reply:
> I haven't looked into the roles of all these RPM build directories. Is there a particular reason for choosing the `BUILDROOT` directory over some other like `BUILD` or `%{buildroot}` -- or arbitrary?

`%{buildroot}` is the installation destination, so we don't want to write temp stuff there.  Beyond that, it doesn't really matter.  With what I've got now, running under mock, the results look like so:

```
clumens@spire ~/src/pacemaker/rpm main|✚1⚑1❯ ls /var/lib/mock/fedora-41-x86_64/root/builddir/build/BUILD/pacemaker-3.0.0-build 
BUILDROOT/  SPECPARTS/  cts-cli.crm_mon.zn4d4o3q  pacemaker-fe199ade4c/
```

Reply:
> Fabio also suggested -- and it seems like a reasonable idea -- updating `cts-cli.in` itself to default to something other than `/tmp`, if `TMPDIR` is not set. Something in the PWD or source dir or whatever; something that we "own."

We're using the tempfile module to handle this at the moment, I would want to check everything it looks at before deciding to use our own default.  That's also `TEMP` and `TMP`.

Reply:
> I would want to check everything it looks at before deciding to use our own default

Within `cts-cli.in`, we could set the `TMPDIR` environment variable (to what we want the default to be) if it's not already set. That would have the same effect, presumably without having to think about corner cases since it would behave the same way as the setting `TMPDIR` before we run the script.